lib/cds_importer/entity_mapper.rb

```diff
@@ -57,8 +59,14 @@ def save_record(record)
       nil
     end
 
+    def remove_excluded_geographical_areas!
+      return if xml_node['sid'].blank?
```

Comment:
Is this defensive or have you actually seen a blank sid for the Measure?

Reply:
Just curious

Reply:
exactly right